Content.Client/Lobby/LobbyState.cs

```diff
@@ -49,7 +52,18 @@ protected override void Startup()
 
             _voteManager.SetPopupContainer(Lobby.VoteContainer);
             LayoutContainer.SetAnchorPreset(Lobby, LayoutContainer.LayoutPreset.Wide);
-            Lobby.ServerName.Text = _baseClient.GameInfo?.ServerName; //The eye of refactor gazes upon you...
+
+            var lobbyNameCvar = _cfg.GetCVar(CCVars.ServerLobbyName);
+            var serverName = _baseClient.GameInfo?.ServerName ?? "";
```

Comment:
If I recall correctly, both are fine, though I think I see `string.Empty` _slightly_ more.

```suggestion
            var serverName = _baseClient.GameInfo?.ServerName ?? string.Empty;
```

Reply:
I suppose it does look more proper